pinot-core/src/test/java/org/apache/pinot/queries/SerializedBytesQueriesTest.java

```diff
@@ -94,7 +94,7 @@
   // Use non-default compression
   private static final double PERCENTILE_TDIGEST_COMPRESSION = 200;
   // Allow 5% quantile error due to the randomness of TDigest merge
-  private static final double PERCENTILE_TDIGEST_DELTA = 0.05 * Integer.MAX_VALUE;
+  public static final double PERCENTILE_TDIGEST_DELTA = 0.05 * Integer.MAX_VALUE;
```

Comment:
Don't expose this value outside of this test

Reply:
Updated.